spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/tracing/MicrometerTracingAutoConfigurationTests.java

```diff
@@ -167,13 +167,13 @@ DefaultTracingObservationHandler customDefaultTracingObservationHandler() {
 		}
 
 		@Bean
-		HttpServerTracingObservationHandler customHttpServerTracingObservationHandler() {
-			return mock(HttpServerTracingObservationHandler.class);
+		PropagatingReceiverTracingObservationHandler customPropagatingReceiverTracingObservationHandler() {
```

Comment:
GH does not let me comment it, but a few lines above the `HttpServerHandler` and `HttpClientHandler` beans are not used but a `Propagator` bean is missing.